nuclear_simulator/data_gen/core/maintenance_tuning_framework.py

```
@@ -0,0 +1,565 @@
+        (self.output_dir / "configs").mkdir(exist_ok=True)
+        (self.output_dir / "results").mkdir(exist_ok=True)
+        (self.output_dir / "data").mkdir(exist_ok=True)
+        (self.output_dir / "reports").mkdir(exist_ok=True)
+        
+        # Initialize components
+        self.composer = ComprehensiveComposer()
+        self.ic_optimizer = ICOptimizer(output_dir=str(self.output_dir), verbose=verbose)
+        self.results_collection = ValidationResultsCollection()
+        
+        if self.verbose:
+            print(f"🔧 MaintenanceTuningFramework initialized")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)